demo/pages/Paper.py

```diff
@@ -0,0 +1,37 @@
+import dash_material_ui as mui
+import dash
+from dash import html, Input, Output, callback
+
+from util import create_component_item
+
+dash.register_page(__name__)
+
+# Paper component layout with different elevation examples
+layout = html.Div(
+    [
+        mui.Grid(
+            [
+                create_component_item(
+                    f"Paper Elevation {elevation}",
+                    mui.Paper,
+                    {
+                        "id": f"paper-{elevation}",
+                        "children": f"Elevation = {elevation}",
+                        "elevation": elevation,
+                    },
+                    event_name="Elevation",
+                    event_value="elevation"
+                )
+                for elevation in [0, 1, 2, 3, 4, 6, 8, 12, 16, 24]
+            ]
+        ),
+    ]
```

Comment:
```suggestion
        html.Div(id="output-div-paper"),
    ]
```

I think it's missing a div, without it I have this error:
<img width="630" alt="image" src="https://github.com/user-attachments/assets/545ee43b-e8fc-4637-b3c8-b350ac394408">